Library/Homebrew/test/support/helper/env.rb

```diff
@@ -1,14 +1,20 @@
 module Test
   module Helper
     module Env
+      def copy_env
+        ENV.to_hash
+      end
+
+      def restore_env(env)
+        ENV.replace(env)
+      end
+
       def with_environment(partial_env)
```

Comment:
> @reitermarkus It's used a few times in the cask tests. Do they automatically reset ENV like the Homebrew tests now do, or would I have to add that?

They do not, but I have a branch, if you want to cherry-pick that commit: https://github.com/reitermarkus/brew/commit/c7c2482caf4cb35e179a52c3171e11f05431a5bb